src/app/forms/components/base-control/base-control.component.ts

```diff
@@ -23,6 +23,7 @@ export class BaseControlComponent implements ControlValueAccessor, AfterContentI
   @Input() width?: FormNodeWidth;
   @Input() viewType: FormNodeViewTypes = FormNodeViewTypes.STRING;
   @Input() noBottomMargin = false;
+  @Input() warning?: string;
```

Comment:
```suggestion
  @Input() warning: string | null = null;
```
You can probably remove the getter if you do this